### DISCUSSION TITLE
Possible to configure the cmp width and positioning?

### DISCUSSION THREAD

**Discussion by JustBarnt** [hrsh7th/nvim-cmp#1965] [Q&A]
Is it possible to configure cmp's width, height, and positioning?

My question stems from not wanting to use Noice `nui` popup because it does not live update itself like cmp does. But currently I cannot seem to find a way to move or adjust the width of cmps width and positioning. 

The best I can get is something like this.
![image](https://github.com/hrsh7th/nvim-cmp/assets/85238383/50970ac7-5a21-4972-9ce9-3a99e898bb36)

**Comment by Shougo** [comment]
For width, you can see this.

https://github.com/hrsh7th/nvim-cmp/issues/980
